src/transformers/models/wav2vec2/processing_wav2vec2.py

```diff
@@ -102,7 +105,22 @@ def from_pretrained(cls, pretrained_model_name_or_path, **kwargs):
                 :class:`~transformers.PreTrainedTokenizer`
         """
         feature_extractor = Wav2Vec2FeatureExtractor.from_pretrained(pretrained_model_name_or_path, **kwargs)
-        tokenizer = Wav2Vec2CTCTokenizer.from_pretrained(pretrained_model_name_or_path, **kwargs)
+
+        # load generic `AutoTokenizer`
+        # need fallback here for backward compatibility in case processor is
+        # loaded from just a tokenizer file that does not have a `tokenizer_class` attribute
+        # behavior should be deprecated in major future release
+        try:
```

Comment:
@sgugger @LysandreJik 

As discussed I think a Wav2Vec2Processor should have the following characteristic:

- Load only a Wav2Vec2FeatureExtractor
- BUT be able to load any tokenizer as Wav2Vec2 is **always** the encoder of a multi-modal model

Reply:
OK!